v2/pkg/nodenet/pod.go

```diff
@@ -218,6 +218,11 @@ func (pn *podNetwork) Setup(nsPath, podName, podNS string, conf *PodNetConf, hoo
 			return fmt.Errorf("netlink: failed to get veth link for container: %w", err)
 		}
 
+		if err := netlink.LinkSetUp(cLink); err != nil {
```

Comment:
Please see https://github.com/containernetworking/plugins/issues/777, we need to run a link setup after getting it.